src/lib/exporter/frameRenderer.ts

```
@@ -94,23 +122,29 @@ export class FrameRenderer {
     // Create canvas for rendering
-    const canvas = document.createElement('canvas');
+    const canvas = document.createElement("canvas");
     canvas.width = this.config.width;
     canvas.height = this.config.height;
-    
+
     // Try to set colorSpace if supported (may not be available on all platforms)
     try {
-      if (canvas && 'colorSpace' in canvas) {
+      if (canvas && "colorSpace" in canvas) {
         // @ts-ignore
```

Biome (2.4.6)

[error] Unsafe use of the `@ts-ignore` directive found in this comment.

(lint/suspicious/noTsIgnore)